pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -851,6 +851,27 @@ self: super: {
   # test suite requires git and does a bunch of git operations
   restless-git = dontCheck super.restless-git;
 
+  sensei = let
+    hspec = self.hspec_2_9_7.override {
+      inherit hspec-core hspec-discover;
+    };
+    hspec-core = self.hspec-core_2_9_7.override {
+      inherit hspec-meta;
+    };
+    hspec-discover = self.hspec-discover_2_9_7.override {
+      inherit hspec-meta;
+    };
+    hspec-meta = self.hspec-meta_2_9_3;
+    hspec-wai = self.hspec-wai.override {
+      inherit hspec hspec-core;
+    };
+  in overrideCabal (drv: {
+    testHaskellDepends = drv.testHaskellDepends or [] ++ [ hspec-meta ];
+    executableSystemDepends = drv.executableSystemDepends or [] ++ [ pkgs.git ];
```

Comment:
I think this should probably be `testSystemDepends`, since `git` is needed when running tests, right?  (If `git` also needs to be available at runtime, then you'll likely need something like `wrapProgram`).

Reply:
`git` is required on the `PATH` both when running the tests and the binary. It will crash otherwise. I agree though, I'll move it to just a test dependency, then rely on users at runtime.

I believe `testToolDepends` would work better here since we're not *linking* with `git`, we just need it on the `PATH`.

Reply:
> I believe testToolDepends would work better here since we're not linking with git, we just need it on the PATH.

Ah, good point, I think you're correct about this.